sdk/core/Azure.Core.Experimental/src/JsonData.cs

```diff
@@ -116,40 +129,6 @@ private JsonData(JsonElement element)
             InitFromElement(element);
         }
 
-        private JsonData(IEnumerable<KeyValuePair<string, JsonData>> properties)
-        {
-            _kind = JsonValueKind.Object;
-            _objectRepresentation = new Dictionary<string, JsonData>();
-            foreach (var property in properties)
-            {
-                if (property.Value == null)
-                {
-                    _objectRepresentation[property.Key] = new JsonData((object?)null);
-                }
-                else
-                {
-                    _objectRepresentation[property.Key] = property.Value;
-                }
-            }
-        }
-
-        private JsonData(IEnumerable<JsonData> array)
-        {
-            _kind = JsonValueKind.Array;
-            _arrayRepresentation = new List<JsonData>();
-            foreach (var item in array)
-            {
-                if (item == null)
-                {
-                    _arrayRepresentation.Add(new JsonData((object?)null));
-                }
-                else
-                {
-                    _arrayRepresentation.Add(item);
-                }
-            }
-        }
-
         private void InitFromElement(JsonElement element)
```

Comment:
This approach to creating the type is a clear performance concern, and will be addressed in this or a later PR.